x-pack/plugins/data_visualizer/public/application/common/components/top_values/top_values.tsx

```diff
@@ -154,9 +147,9 @@ export const TopValues: FC<Props> = ({ stats, fieldFormat, barColor, compressed,
             <EuiText size="xs" textAlign={'center'}>
               <FormattedMessage
                 id="xpack.dataVisualizer.dataGrid.field.topValues.calculatedFromSampleDescription"
-                defaultMessage="Calculated from sample of {topValuesSamplerShardSize} documents per shard"
+                defaultMessage="Calculated from sample of {topValuesSampleSize} documents"
```

Comment:
Is this message still needed? If random sampler isn't turned on, then there is no 'sampling' is there? And if random sampling is being used, it mirrors the doc count displayed at the top.

<img width="265" alt="image" src="https://user-images.githubusercontent.com/7405507/186443454-44b6db13-ba7f-45a6-89a8-ec29a9aa68ab.png">